rj_gameplay/rj_gameplay/basic_play_selector.py

```diff
@@ -77,7 +77,7 @@ def __init__(self):
 
     def select(
         self, world_state: rc.WorldState
-    ) -> Tuple[situation.ISituation, stp.play.IPlay]:
+    ) -> Tuple[Optional[situation.ISituation], stp.play.IPlay]:
```

Comment:
`Error: rj_gameplay/rj_gameplay/gameplay_node.py:35: error: Incompatible return value type (got "Tuple[None, PenaltyDefense]", expected "Tuple[ISituation, IPlay[Any]]")`

For TestPlaySelector the situation returned is None, so I changed the type to Optional to reflect this.